### ISSUE TITLE
about dataloader

### ISSUE THREAD

**Issue by asw91666** [karfly/learnable-triangulation-pytorch#136]
Hi! thank you for amazing contribution.
I wonder why do you use islice() during training.

https://github.com/karfly/learnable-triangulation-pytorch/blob/9d1a26ea893a513bdff55f30ecbfd2ca8217bf5d/train.py#L174

thank you

**Comment by shrubb** [comment]
Hi, thanks. I believe it was used to limit the number of iterations per epoch. For example, if your metric is "accuracy after 20 epochs" and if you have two datasets where one of them is twice bigger than the other, that metric will be inconsistent. Because on a larger dataset, 20 epochs will mean twice more training steps.

**Comment by asw91666** [comment]
then didn't you use all frame of data/human36m/processed files when you train ?

**Comment by shrubb** [comment]
Sometimes yes, sometimes not (like in experiments "without damaged actions"). Also, we used the CMU Panoptic dataset.

**Comment by asw91666** [comment]
Do you mean you use H36m and CMU Panoptic jointly for learning, and testing on h36m?

**Comment by shrubb** [comment]
> you use H36m and CMU Panoptic jointly for learning

As far as I remember, no. You should check the paper, we've explained everything there in detail.

**Comment by asw91666** [comment]
OK Thank you very much !
